src/utils/searchUtils.ts

```diff
@@ -94,23 +97,11 @@ export function isInteractionCompatible({
   interactionType,
   functionName,
 }: {
-  contractType: ContractType;
-  interactionType: AntInteraction | RegistryInteraction;
+  contractType: CONTRACT_TYPES;
+  interactionType: InteractionTypeName;
   functionName: string;
 }) {
   try {
-    if (contractType === CONTRACT_TYPES.UNKNOWN) {
-      throw new Error(`contract type of ${contractType} is not compatible`);
-    }
-    if (
-      interactionType === ANT_INTERACTION_TYPES.UNKNOWN ||
-      interactionType === REGISTRY_INTERACTION_TYPES.UNKNOWN
-    ) {
-      throw new Error(
-        `interaction type of ${interactionType} is not compatible`,
-      );
-    }
-
```

Comment:
The whole idea of having explicit types was to remove this part here and the one below lol